test/integration/golden_response/staging/get_triples/stat_var_svobs.json

```diff
@@ -508,6 +538,16 @@
       ],
       "provenanceId": "dc/d7tbsb1"
     },
+    {
+      "subjectId": "Count_Person_Female",
+      "predicate": "constraintProperties",
+      "objectId": "memberOf",
+      "objectName": "memberOf",
+      "objectTypes": [
```

Comment:
@pradh do we expect "memberOf" to be constraint prop?

Reply:
Good catch. I need to add memberOf to one more list to avoid this...

Reply:
thanks